extension-manifest-v3/src/store/tab-stats.js

```diff
@@ -26,6 +28,10 @@ const Tracker = {
 const Stats = {
   domain: '',
   trackers: [Tracker],
+  trackersBlocked: ({ trackers }) =>
```

Comment:
Move `trackers...` to the end of the computer properties list (after `categories`).